blog-posts/emails/article.md

```diff
@@ -0,0 +1,331 @@
+---
+published: false
+title: 'Sending emails with AWS SES: three simple steps from zero to hero! 🦸‍♂️'
+cover_image: https://raw.githubusercontent.com/pchol22/kumo-articles/master/blog-posts/emails/assets/cover.png
+tags: serverless, AWS, SES, Emails
+series:
+canonical_url:
+---
+
+_This article assumes the reader has **really basic** knowledge of SES, like being able to send simple emails using SES and Lambda or to verify an identity_
+
+As you may already know it, AWS SES offers a great way to send emails with your AWS Serverless app. Coupled with the AWS sdk and Lambda functions, it allows you to programmatically send emails to your users with minimal setup. However, this ease of use comes at the price of potential mistakes and pitfalls that can happen at every step of your coding journey. Let me guide you through **three** of these issues and help you design beautiful emails that will always end up in your users mailbox.
+
+## 🚧 🚧 🚧 Three obstacles on your road to clean email communication
+
+Few months ago, I went through the implementation of transactional emails sending on the project I was working on, and identified three major pitfalls that every developer should be aware of.
+
+Using only **minimal configuration** to send your emails:
+
+- 🚧 You will be prevented from sending messages if your sender reputation goes down.
```

Comment:
"**Using only** ... to send your emails, you will be ..., **if** your sender ...", there are two conditions that make your message harder to read. Is this what you mean: ""**Using only** ... to send your emails, you will **probably** be ..., **because** your sender **will [probably?]** ..."